### PR TITLE
ref: alt enter

### PR DESCRIPTION
**Pull request by bruno-garcia**
I was bored

### CODE REVIEW

File: src/Sentry.AspNetCore.Grpc/SentryBuilderExtensions.cs [hunk 1]
```diff
@@ -24,5 +28,29 @@ public static ISentryBuilder AddGrpc(this ISentryBuilder builder)
...
+            private static readonly SdkVersion NameAndVersion
+                = typeof(SentryGrpcInterceptor).Assembly.GetNameAndVersion();
+
+            private static readonly string ProtocolPackageName = "nuget:" + NameAndVersion.Name;
+            private const string SdkName = "sentry.dotnet.aspnetcore.grpc";
+
+            public SentryEvent Process(SentryEvent @event)
+            {
+                // Take over the SDK name since this wraps ASP.NET Core
+                @event.Sdk.Name = SdkName;
```
Comment (bruno-garcia) [inline src/Sentry.AspNetCore.Grpc/SentryBuilderExtensions.cs:44]: THe only change that should have side affect. The rest should be small refactors

File: src/Sentry.NLog/SentryTarget.cs [hunk 2]
```diff
@@ -310,7 +305,7 @@ protected override void Write(LogEventInfo logEvent)
 
             var hub = HubAccessor();
 
-            if (hub?.IsEnabled != true)
+            if (hub.IsEnabled != true)
```
Comment (bruno-garcia) [inline src/Sentry.NLog/SentryTarget.cs:308]:
```suggestion
            if (!hub.IsEnabled)
```

File: test/Sentry.Tests/ModuleInitializerAttribute.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-#if(!NET5_0)
+#if !NET5_0
```
Comment (bruno-garcia) [inline test/Sentry.Tests/ModuleInitializerAttribute.cs:1]: when .NET 6 comes around this will be a problem

**Review by Tyrrrz — APPROVED** [review]

**Comment by bruno-garcia** [comment]
> D:\a\sentry-dotnet\sentry-dotnet\test\Sentry.Tests\Sentry.Tests.csproj : error NU1101: Unable to find package Microsoft.WindowsDesktop.App.Ref. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\sentry-dotnet\sentry-dotnet\Sentry.sln]


What's up with GH Actions Windows .. jeez

File: test/Sentry.Tests/ModuleInitializerAttribute.cs (hunk 3 above)
Comment (arush162) [reply test/Sentry.Tests/ModuleInitializerAttribute.cs:1]:
```suggestion
#if !NET5_0
```

Comment (arush162) [reply test/Sentry.Tests/ModuleInitializerAttribute.cs:1]: > when .NET 6 comes around this will be a problem

I know that happens to ma some times

**Comment by arush162** [comment]
usr/bin/docker run --name e4b82a2899f0814bbdb3173f35db19f668_5a297a --label 5588e4 --workdir /github/workspace --rm -e INPUT_API-TOKEN -e INPUT_MIN-SCORE -e INPUT_UPLOAD-TO-CODE-SCANNING -e INPUT_GITHUB-TOKEN -e INPUT_COLLECTION-NAME -e INPUT_PLATFORM-URL -e INPUT_LOG-LEVEL -e INPUT_IGNORE-FAILURES -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/animated-funicular/animated-funicular":"/github/workspace" 5588e4:b82a2899f0814bbdb3173f35db19f668



this one failed

**Comment by arush162** [comment]
@bruno-garcia  it is kinda tough

**Pull request merged**